### PR TITLE
Add calculator UI components

### PR DESCRIPTION
**Pull request by Metaverse-Nabeel**
### The following changes have been made to this PR.

- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called `components`.
- Inside components, created a n0ew Calculator.js file.
- In `Calculator.js`, create a React component `CalculatorBtns` that matches this design:
- Implemented the design only (JSX and CSS)
- Used functional components
- Created at least 1 child component
- Tested it in the browser; it should render without problems.
- Updated Readme file
- Corrected Linter errors

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +0,0 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
```
Comment (TedLivist) [inline src/App.test.js:8]: - [x] Kindly remove this file and the other ones marked in the image below as they are boilerplate files that won't be used.

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Metaverse-Nabeel,

Good job so far!
✔️ Nice UI design
✔️ Use of functional component for calculator
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.test.js (hunk 1 above)
Comment (Metaverse-Nabeel) [reply src/App.test.js:8]: Hello @TedLivist 

Thank you for your feedback. I shall make the required changes and ask for a review again.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Metaverse-Nabeel,

✔️ Nice UI design

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**